src/clr-angular/utils/popover/providers/popover-toggle.service.ts

```diff
@@ -47,6 +47,10 @@ export class ClrPopoverToggleService {
     return this._open;
   }
 
+  get originalEvent(): Event {
```

Comment:
Is this used?

Reply:
Yes, in `abstract-popover` and `dropdown-focus-handler.service`